modules/ROOT/pages/proc_assembling-a-pipeline.adoc

```diff
@@ -114,13 +114,13 @@ spec:
 +
 This `Pipeline` performs the following tasks:
 
-* Clones the source code of the frontend application from the `api-repo` Git repository and the backend application from the `ui-reporesource` Git repository.
-* Builds the container image for the frontend using the `s2i-python-3` task that generates a Dockerfile for the application using Source-to-Image (S2I). 
+* Clones the source code of the frontend application from the `ui-repo` Git repository and the backend application from the `api-repo resource` Git repository.
```

Comment:
Wonders, isnt it it the APi first and UI later as an order of pipeline execution? just wondering flow wise

Reply:
As I understood, `ui-repo` is frontend, and `api-repo` is backend. It was written the other way ( `api-repo` for frontend, and` ui-repo` or backend) in the previous doc version. I just correct the repo names to match with their functionality. As suggested, I will change the order here (backend followed by frontend) to match the pipeline workflow.

Reply:
:+1: